sql/catalyst/src/main/scala/org/apache/spark/sql/errors/QueryCompilationErrors.scala

```diff
@@ -1344,6 +1344,14 @@ private[sql] object QueryCompilationErrors extends QueryErrorsBase {
     tableDoesNotSupportError("truncates", table)
   }
 
+  def tableDoesNotSupportUpsertsError(table: String): Throwable = {
+    new AnalysisException(
+      errorClass = "_LEGACY_ERROR_TEMP_1121",
+      messageParameters = Map(
+        "cmd" -> "upserts",
+        "table" -> table))
```

Comment:
`tableDoesNotSupportError("upserts", table)`

Reply:
Method `tableDoesNotSupportError` requires a `Table`, where I have only a table name string.

Reply:
I see.

Reply:
So we can refactor
`private def tableDoesNotSupportError(cmd: String, table: Table): Throwable`
to
`def tableDoesNotSupportError(cmd: String, tableName: String): Throwable` and update the callers.

Reply:
Good idea, done in 085f9af8.